onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorLayerNormalization.cpp

```diff
@@ -6,6 +6,7 @@
 namespace Dml
 {
 
+template <bool simplified>
```

Comment:
Hmm, let's not bloat up an entire sizeable class based on a single template parameter. Rather than hoping the compiler is good enough to deduplicate identical COMDAT's, we can just do what activation does.

onnxruntime\core\providers\dml\DmlExecutionProvider\src\Operators\DmlOperatorActivation.cpp
```
// A specific type of operation for registration.
template <DML_OPERATOR_TYPE OperatorType>
class DmlOperatorActivationTemplate : public DmlOperatorActivation
{
public:
    DmlOperatorActivationTemplate(const MLOperatorKernelCreationContext& kernelCreationContext)
    :   DmlOperatorActivation(kernelCreationContext, OperatorType)
    {
    }
};

DML_OP_DEFINE_CREATION_FUNCTION(Sigmoid,             DmlOperatorActivationTemplate<DML_OPERATOR_ACTIVATION_SIGMOID>);
DML_OP_DEFINE_CREATION_FUNCTION(HardSigmoid,         DmlOperatorActivationTemplate<DML_OPERATOR_ACTIVATION_HARD_SIGMOID>);
DML_OP_DEFINE_CREATION_FUNCTION(Tanh,                DmlOperatorActivationTemplate<DML_OPERATOR_ACTIVATION_TANH>);
DML_OP_DEFINE_CREATION_FUNCTION(ScaledTanh,          DmlOperatorActivationTemplate<DML_OPERATOR_ACTIVATION_SCALED_TANH>);
DML_OP_DEFINE_CREATION_FUNCTION(Relu,                DmlOperatorActivationTemplate<DML_OPERATOR_ACTIVATION_RELU>);
```